openlibrary/macros/SearchResultsWork.html

```diff
@@ -25,12 +25,12 @@
 
   <span class="details">
       <span class="resultTitle">
-         <h3 class="booktitle">
+         <span class="h3 booktitle">
```

Comment:
please keep this as an h3.
The title should remain a heading as it describes the heading of a book.

If anything should change `span.resultTitle` and `span.details` should become `div`.